INSTALL.md

```diff
@@ -60,7 +60,7 @@ make dev-stop
 
 #### Ruby version
 
-The project supports **Ruby 2.4.x**.
+The project supports **Ruby 2.5.x**.
```

Comment:
Maybe just say both?

Reply:
It is not a very strong opinion but I think that we shouldn't say that we support 2.4 after the EOL so we can make our life easier. Because if it is officially unsupported and we have to maintain it ourselves and ensure by ourselves that it keeps working... that sounds problematic.

Reply:
I agree with @Martouta